app/views/content/subjects/physics/_article.html.erb

```diff
@@ -0,0 +1,162 @@
+        14 to 16 year olds (key stage 4):
+      </p>
+       <ul class="vertical-tags">
+        <li class="tag">Energy, forces and wave motion</li>
+        <li class="tag">Electricity, magnetism and electromagnetism</li>
+        <li class="tag">The structure of matter</li>
+        <li class="tag">Atomic structure and space physics</li>
+      </ul>
+      <p>
+        16 to 18 year olds (key stage 5):
+      </p>
+       <ul class="vertical-tags">
+        <li class="tag">Fields, vectors and scalars</li>
+        <li class="tag">Mechanics and mechanical properties of matter</li>
+        <li class="tag">Electric circuits and waves</li>
+        <li class="tag">Matter, quantum and nuclear physics</li>
+      </ul>
+    <p>
+      <a href="https://www.gov.uk/government/publications/national-curriculum-in-england-science-programmes-of-study/national-curriculum-in-england-science-programmes-of-study">Read the national curriculum for more information about physics lessons</a>.
+    </p>
+
+    <p>You can also search for past exam papers online to see the type of questions you'll be teaching about.</p>
+
+    <p>
+      You may also get the opportunity to teach other subjects, particularly for 11 to 16 year olds and in some cases for post-16. This will vary depending on individual schools.
+    </p>
+    <p><a href="https://www.ogdentrust.com/teacher-support/training-to-be-a-teacher/">Find out more about becoming a physics teacher from the Ogden Trust</a>, a charitable trust which promotes the teaching and learning of physics. </p>
+  </section>
+   <section class="col col-720">
+  <%= render Content::QuoteComponent.new(
+    text: "It's inspiring getting 'wow, that's how it works' from pupils as they grasp a subject that they thought was hard.",
+    name: "Jane Doran, physics and robotics teacher, West Midlands",
+    background: "grey",
+    large: true
+  ) %>
+</section>
+
+ <section class="col col-space-s col-content-max">
+    <%= render TextBlockWithOneImageComponent.new(
+      heading: "Next steps into teacher training",
+      colour: "yellow",
+      image_path: "static/content/hero-images/0006.jpg",
+      image_alt: "Classroom full of pupils.",
+    ) %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Help to become a physics teacher</h3>
+
+    <p>
+      Our experienced teacher training advisers can guide you through everything you need to know to apply for physics teacher training, free of charge. They can help you to:
+    </p>
+    <ul>
+      <li>understand more about physics courses</li>
+      <li>answer questions about fees and funding</li>
+      <li>make an application for a physics course</li>
+    </ul>
+    <p><a href="/teacher-training-advisers">Find out about advisers</a>.</p>
+
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Check your qualifications</h3>
+    <p>
+      To teach physics in England, you'll need a bachelor's degree, but it does not have to be in physics.
```

Comment:
I think this is a bit misleading now - could you infer from this that you just need a degree and not QTS @EllieNodder @philspray ?

```suggestion
      To train to teach physics in England, you'll need a bachelor's degree, but it does not have to be in physics.
```